scripts/offline-install.sh

```diff
@@ -49,22 +58,10 @@ function main {
     # Prepare install
     pip3 download esrally=="${RALLY_VERSION}" --dest "${ABSOLUTE_DOWNLOAD_BIN_DIR}"
 
-    # grab all license files
-    echo "Downloading license files for all dependencies"
-    # direct dependencies
-    curl --silent -o "${ABSOLUTE_DOWNLOAD_BIN_DIR}/certifi-LICENSE.txt" https://raw.githubusercontent.com/certifi/python-certifi/master/LICENSE
-    curl --silent -o "${ABSOLUTE_DOWNLOAD_BIN_DIR}/elasticsearch-LICENSE.txt" https://raw.githubusercontent.com/elastic/elasticsearch-py/master/LICENSE
-    curl --silent -o "${ABSOLUTE_DOWNLOAD_BIN_DIR}/jinja2-LICENSE.txt" https://raw.githubusercontent.com/pallets/jinja/master/LICENSE
-    curl --silent -o "${ABSOLUTE_DOWNLOAD_BIN_DIR}/jsonschema-LICENSE.txt" https://raw.githubusercontent.com/Julian/jsonschema/master/COPYING
-    curl --silent -o "${ABSOLUTE_DOWNLOAD_BIN_DIR}/psutil-LICENSE.txt" https://raw.githubusercontent.com/giampaolo/psutil/master/LICENSE
-    curl --silent -o "${ABSOLUTE_DOWNLOAD_BIN_DIR}/py-cpuinfo-LICENSE.txt" https://raw.githubusercontent.com/workhorsy/py-cpuinfo/master/LICENSE
-    curl --silent -o "${ABSOLUTE_DOWNLOAD_BIN_DIR}/tabulate-LICENSE.txt" https://bitbucket.org/astanin/python-tabulate/raw/03182bf9b8a2becbc54d17aa7e3e7dfed072c5f5/LICENSE
-    curl --silent -o "${ABSOLUTE_DOWNLOAD_BIN_DIR}/thespian-LICENSE.txt" https://raw.githubusercontent.com/kquick/Thespian/master/LICENSE.txt
-    # transitive dependencies
-    # Jinja2 -> Markupsafe
-    curl --silent -o "${ABSOLUTE_DOWNLOAD_BIN_DIR}/markupsafe-LICENSE.txt" https://raw.githubusercontent.com/pallets/markupsafe/master/LICENSE
-    # elasticsearch -> urllib3
-    curl --silent -o "${ABSOLUTE_DOWNLOAD_BIN_DIR}/urllib3-LICENSE.txt" https://raw.githubusercontent.com/shazow/urllib3/master/LICENSE.txt
+
+    echo "Preparing NOTICE file"
+    __NOTICE_OUTPUT_FILE="${ABSOLUTE_DOWNLOAD_BIN_DIR}/NOTICE.txt"
+    source "${SCRIPT_SRC_HOME}/../create-notice.sh"
```

Comment:
Referencing a script from a different directory is not nice. I think we should move our build-related scripts to a dedicated directory. However, I wanted to keep the scope of this PR as small as possible.